README.md

```diff
@@ -1,33 +1,93 @@
 # PiCraft
 
-### Control a Minecraft world on a Raspberry Pi from Julia
+### Control a Minecraft world on the Minecraft: Pi or Java edition from Julia
 
 ## Installation and Usage
+
+### Get Minecraft
+
+- Get the Minecraft: Pi edition [here](https://minecraft.net/en-us/edition/pi/). It's free!
```

Comment:
Maybe mention that is installed in the default image for the Rasbian linux distribution for the Pi

Reply:
It's not on Raspbian Jessie and Pi version has been *officially* discontinued. Source: https://www.reddit.com/r/MCPE/comments/42g9cr/is_minecraft_pi_edition_no_longer_being_developed/cza7seq/?context=3&st=jfy7cy9x&sh=12b0b092

Reply:
Although they did say it will come with every Raspbian distro I wouldn't personally count on it.

Reply:
I actually emulated Raspbian Jessie using VirtualBox and this mc version needs an ARM processor, need to do some work to get it to run with an x86 machine. Maybe that's why it didn't show up.

Reply:
Yes, while the new minecraft platform edition code does not support the Pi, the existing codebase will continue to be available. Its an important part of the Rasberry Pi ecosystem.